apps/src/aiTutor/views/AssistantMessageFeedbackDetails.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useState} from 'react';
+
+import Button, {buttonColors} from '@cdo/apps/componentLibrary/button/Button';
+
+import {saveFeedback, FeedbackData} from '../interactionsApi';
+
+interface AssistantMessageFeedbackDetailsProps {
+  messageId: number | undefined;
+  feedbackData: FeedbackData;
+}
+
+const AssistantMessageFeedbackDetails: React.FC<
+  AssistantMessageFeedbackDetailsProps
+> = ({feedbackData, messageId}) => {
+  const [feedbackDetails, setFeedbackDetails] = useState<string>();
+
+  const handleFeedbackSubmission = async (
+    feedbackData: FeedbackData,
+    feedbackDetails: string | undefined,
+    messageId?: number
+  ) => {
+    if (!messageId) {
+      return;
+    }
+
+    const feedback = {
+      thumbsUp: feedbackData.thumbsUp,
+      thumbsDown: feedbackData.thumbsDown,
+      details: feedbackDetails,
+    };
+
+    try {
+      await saveFeedback(messageId, feedback);
+      setFeedbackDetails('');
+    } catch (error) {
+      console.log('Error saving feedback.');
+    }
+  };
+
+  return (
```

Comment:
- [ ] adjust styling of this component to be in-line and less intrusive (e.g. probably smaller)